applications/ULFapplication/custom_processes/mass_calculate_process.h

```diff
@@ -165,7 +165,7 @@ class MassCalculateProcess
             im->Calculate(NODAL_MASS,dummy,proc_info);
         }
 
-        KRATOS_WATCH("Execute of Mass Calulate Process");
+        KRATOS_WATCH("Execute of Mass Calculate Process");
```

Comment:
`KRATOS_WATCH` is a glorified `std::cout` that people mostly use for debugging. It should never have been committed so we can get rid of them.

```suggestion
```